node/core/dispute-coordinator/src/lib.rs

```diff
@@ -25,9 +25,6 @@
 //! another node, this will trigger the dispute participation subsystem to recover and validate the block and call
 //! back to this subsystem.
 
-#![forbid(unused_must_use)]
-#![forbid(unused_crate_dependencies)]
-
```

Comment:
@drahnr Is this ok? Otherwise we would need to feature gate most of the deps in this crate, which imo should be done in another PR

Reply:
I just introduced it as part of this PR, so it's alright, removed two deps :)